gui/screens/settings.py

```diff
@@ -405,6 +424,78 @@ def is_testnet_mode_toggle_checked(self):
+    def __init__(self):
+        super(EditNetworkSettings, self).__init__()
+        self._live_network_tab = Button('editNetworkLiveButton')
+        self._test_network_tab = Button('editNetworkTestButton')
+        self._network_name = TextEdit('editNetworkNameInput')
+        self._network_short_name = TextEdit('editNetworkShortNameInput')
+        self._network_chaid_id = TextEdit('editNetworkChainIdInput')
+        self._network_native_token_symbol = TextEdit('editNetworkSymbolInput')
+        self._network_main_json_rpc_url = TextEdit('editNetworkMainRpcInput')
+        self._network_failover_json_rpc_url = TextEdit('editNetworkFailoverRpcUrlInput')
+        self._network_block_explorer = TextEdit('editNetworkExplorerInput')
+        self._network_acknowledgment_checkbox = CheckBox('editNetworkAknowledgmentCheckbox')
+        self._network_revert_to_default = Button('editNetworkRevertButton')
+        self._network_save_changes = Button('editNetworkSaveButton')
+        self._network_edit_view_back_button = Button('main_toolBar_back_button')
+        self._network_edit_scroll = Scroll('settingsContentBaseScrollView_Flickable')
+
+    @allure.step('Verify elements for the edit network view')
+    def check_available_elements_on_edit_view(self):
+        assert self._network_edit_view_back_button.exists, f"Back button is not present"
+        assert self._live_network_tab.exists, f"Live tab is not present"
+        assert self._test_network_tab.exists, f"Test tab is not present"
+        assert self._network_name.exists, f"Network name input field is not present"
+        assert self._network_short_name.exists, f"Short name input field is not present"
+        assert self._network_chaid_id.exists, f"Chaid Id input field is not present"
+        assert self._network_native_token_symbol.exists, f"Native token symbol input field is not present"
+        assert self._network_main_json_rpc_url.exists, f"Main JSON RPC URL input field is not present"
+        assert self._network_failover_json_rpc_url.exists, f"Failover JSON RPC URL input field is not present"
+        assert self._network_block_explorer.exists, f"Block explorer input field is not present"
+
+        self._network_edit_scroll.vertical_down_to(self._network_acknowledgment_checkbox)
+        assert driver.waitFor(lambda: self._network_acknowledgment_checkbox.exists,
+                              configs.timeouts.UI_LOAD_TIMEOUT_MSEC)
+
+        self._network_edit_scroll.vertical_down_to(self._network_revert_to_default)
+        assert driver.waitFor(lambda: self._network_revert_to_default.exists,
+                              configs.timeouts.UI_LOAD_TIMEOUT_MSEC)
+
+        assert self._network_save_changes.exists
+
+    @allure.step('Edit the rpc input field')
+    def edit_network_main_json_rpc_url_input(self, test_value):
+        self._network_main_json_rpc_url.click()
+        self._network_main_json_rpc_url.clear()
+        self._network_main_json_rpc_url.text = test_value
```

Comment:
You can delete 470-471 and also 476-477, because method text is enough - it clears field and put the value :)

Reply:
fixed!